src/test/resources/yaml/a2.yaml

```diff
@@ -26,8 +26,7 @@ tests:
     methodSelectors:
       - className: org.testng.internal.XmlMethodSelector
         priority: 1
-      - expression: groups.containsKey("test1")
-        language: beanshell
+      - script: { expression: groups.containsKey("test1"), language: beanshell }
```

Comment:
The yaml format is not supposed to change but some deprecated methods were used by the yaml library.
Could you check how it is possible to customize the yaml serialization?

Reply:
Fixed and pushed.